app/src/main/java/org/fossasia/openevent/app/core/faq/list/viewholder/FaqViewHolder.java

```diff
@@ -5,20 +5,45 @@
 import android.view.View;
 
 import org.fossasia.openevent.app.R;
+import org.fossasia.openevent.app.common.Pipe;
 import org.fossasia.openevent.app.data.models.Faq;
 import org.fossasia.openevent.app.databinding.FaqLayoutBinding;
 
 public class FaqViewHolder extends RecyclerView.ViewHolder {
 
     private final FaqLayoutBinding binding;
+    private Faq faq;
+    private Pipe<Long> longClickAction;
+    private Pipe<Long> clickAction;
```

Comment:
Use `Function`

Reply:
The return type of referenced methods like `toolbarDeleteMode()` and `listItemClickAction` is void. So why is `Function` needed instead of `Pipe` ?

Reply:
Yes, it shouldn't be Function, it should be runnable